storybook/stories/Welcome.mdx

```diff
@@ -0,0 +1,44 @@
+import { CartesianGrid, Line, LineChart, XAxis, YAxis, Tooltip } from '../../src';
+import { pageData as data } from './data/Page';
+
+# Recharts
+A composable charting library built on React components
+<LineChart width={500}
+           height={300}
+           data={data}
+           margin={{
+             top: 5,
+             right: 5,
+             bottom: 5,
+             left: 0,
+           }}>
+  <CartesianGrid stroke="#eee" strokeDasharray="5 5" />
+  <XAxis dataKey="name" />
+  <YAxis />
+  <Line type="monotone" dataKey="uv" stroke="#8884d8" />
+  <Line type="monotone" dataKey="pv" stroke="#82ca9d" />
+  <Tooltip />
+</LineChart>
+
+```typescript jsx
+  <LineChart width={500} height={300} data={data}>
```

Comment:
I think we could indent one space to the left to make it has a better look 
<img width="210" alt="image" src="https://user-images.githubusercontent.com/43896664/222661519-d66e02f3-6bae-4b7e-b77a-e3e993c819e2.png">